### ISSUE TITLE
communication but no control

### ISSUE THREAD

**Issue by v0d0r** [tixi/Domoticz-Tuya-SmartBulb-Plugin#2]
It seems I can communicate with my smarb bulb but I cannot control it. It did briefly work but then stopped again.
Something else interesting I discovered. After disabling the plugin in domoticz and using tuya-cli I immediately get a response via tuya-cli. Re-enable the domoticz plugin then I get no control on the bulb but no response from tuya-cli (assuming this is the 1 connection limit so domoticz must be connect) 
Any suggestions what else to try?

snip tcpdump shows 2 way communication.

18:08:35.844079 IP 192.168.128.7.47150 > 192.168.128.21.6668: Flags [P.], seq 44033:44119, ack 50397, win 65392, length 86
18:08:35.844602 IP 192.168.128.21.6668 > 192.168.128.7.47150: Flags [.], ack 44033, win 5840, length 0
18:08:35.866892 IP 192.168.128.21.6668 > 192.168.128.7.47150: Flags [P.], seq 50397:50690, ack 44119, win 5754, length 293
18:08:35.900859 IP 192.168.128.7.47150 > 192.168.128.21.6668: Flags [P.], seq 44119:44290, ack 50690, win 65392, length 171
##################################3

snip from tuya-cli
tuya-cli get --ip 192.168.128.21 --id 82504753cc50e3146d08 --key ****************** -a
{ devId: '82504753cc50e3146d08',
dps:
{ '1': true,
##################

**Comment by v0d0r** [comment]
Update: From time to time I also see "Async Read Exception: 104, Connection reset by peer"

**Comment by mzy2240** [comment]
Yes same here. Successfully retrieved the key and id by following this link https://github.com/codetheweb/tuyapi/blob/master/docs/SETUP.md, however, Domoticz cannot control or retrieve states from the Tuya devices, and in the log there is an error "Async Read Exception: 104, Connection reset by peer". Any idea on this? @tixi 
BTW, I have a google assistant connecting to the Tuya device, is that the reason?

**Comment by tixi** [comment]
You should have only one connection at a time to the device meaning no app, no google assistant or other while using the plugin

**Comment by Adam-1969** [comment]
Hi Tixi,
I have a Tuya bulb but a version without RGB. I connected it through the method of commandline to find the keys.
Problem is that the bulb does not respond to commands and i hav the obove mentioned " Async Read Exception: 104, Connection reset by peer" error. It looks like there is communication with the bulb when debug is on.
There is no other connection to the bulb, not from the app not from google assistant.

Hope you can help me, thanks in advance
